node_filesystem/templates/bird.cfg.template

```diff
@@ -3,6 +3,14 @@ include "bird_ipam.cfg";
 
 router id {{getenv "IP"}};
 
+{{define "LOGGING"}}
+{{$node_logging_key := printf "/host/%s/LogSeverityScreen" (getenv "HOSTNAME")}}{{if exists $node_logging_key}}{{$logging := getv $node_logging_key}}
+{{if eq $logging "debug"}}  debug all;{{else if ne $logging "none"}}  debug { states };{{end}}
```

Comment:
Just what we currently use for our "info" level logging.  Could perhaps use some more refining based on conversations with Shaun from his scale testing.